README.md

```diff
@@ -4,6 +4,16 @@
 
 The PegaSys Orchestrate library provides convenient access to the PegaSys Orchestrate API from applications written in server-side JavaScript
 
+# Compatibility
+
+| SDK versions        | Orchestrate versions        |
+| ------------------- | --------------------------- |
+| master/HEAD         | (unreleased)                |
+| SDK v2.2.x          | (unreleased)                |
```

Comment:
Instead of saying (unreleased) we can say `Orchestrate v2.1.0-rc4` after all the image is available